baseline-refaster-rules/build.gradle

```diff
@@ -34,4 +34,5 @@ moduleJvmArgs {
             'jdk.compiler/com.sun.tools.javac.tree',
             'jdk.compiler/com.sun.tools.javac.util'
     ]
+    opens = ['jdk.compiler/com.sun.tools.javac.comp']
```

Comment:
For my understanding, why do we need the additional "opens" declaration now? I don't see any new usage in the diff here.

Reply:
This allows this module to pass tests using jdk17. Not required for the build, but initially I tried using jdk17 everywhere, but some presets are more difficult to fix.